script.js

```diff
@@ -1,4 +1,29 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // randomly chooses scissors, paper or stone
+  var useroption =input;
+  var option1 = "scissors";
+  var option2 = "paper";
+  var option3 = "stone";
```

Comment:
The variable names can be improved: the idea is our variable names should be as meaningful as possible, so that even by looking at the variable name I know what's the value stored in the variable.